assets/stylesheets/hootstrap/patterns/_page-header.scss

```diff
@@ -5,21 +5,7 @@
 }
 
 .page-title {
```

Comment:
Given the specific margin on the `.page-subtitle` class, I'd force the `.page-title` class to have a display of `block`